app/workers/push_notification_worker.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "rpush/daemon"
+
+class PushNotificationWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :push_notification, retry: 0
+
+  def perform(notification_id)
+    Rpush.config.push = true
+    Rpush::Daemon.common_init
+    Rpush::Daemon::Synchronizer.sync
+    Rpush::Daemon::AppRunner.enqueue(Rpush::Client::ActiveRecord::Notification.where(id: notification_id))
+    Rpush::Daemon::AppRunner.stop
```

Comment:
hmm...  I wonder if it would be better to put some of these in the sidekiq initializer, e.g.
```ruby
Sidekiq.configure_server do
    Rpush.config.push = true
    Rpush::Daemon.common_init
    Rpush::Daemon::Synchronizer.sync
end

at_exit do
    Rpush::Daemon::AppRunner.stop
end
```

and just leave the enqueue bit in the job.

sounds like these start/stop global instances of some kind which might cause some issues if two of these jobs run at the same time in the same sidekiq process